### ISSUE TITLE
Add to Stackage

### ISSUE THREAD

**Issue by ndmitchell** [haskell/ghcide#144]
Having ghcide in Stackage is super useful for company's that base their package set on Stackage. Could it please be added? I'm happy to be the point person and add it myself, if you wish.

Note that adding it right now might not be the best timing since Stackage just rolled to GHC 8.8 and consequently everything is in flux. But in a few weeks it's a definitely useful step.

**Comment by cocreature** [comment]
Sure, happy to add it once the deps are back in nightly.

**Comment by ndmitchell** [comment]
Everything is now back in nightly.

**Comment by cocreature** [comment]
We’ll need a new release before we can add it to stackage. I was hoping to include a bump to a newer `hie-bios` release in the next release but given that this seems to be taking longer than I initially thought, I’ll probably make one without that.

**Comment by turion** [comment]
Releasing to stackage would be great! Plans on doing that soon?

**Comment by v217** [comment]
For the vast majority of users running a docker container is easier, than getting familiar with Nix. With the vscode extension [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) this has become the [fastest way to get started](https://hmemcpy.com/2020/02/setting-up-a-haskell-development-environment-in-minutes-in-vscode/)!

So until there is a great solution for binaries for stackage, a docker image is maybe the most user-friendly option?

Nix has other problems as well! It [only  works](https://github.com/NixOS/nix/issues/440) with bash, it has [problems with xattrs](https://github.com/NixOS/nix/commit/d798349ede3d6eb6e92a2e4f95f6b2179407ceb9) on [NFS, ZFS, btrfs, etc ...](https://github.com/NixOS/nixpkgs/issues/29778)

**Comment by ndmitchell** [comment]
@v217 - adding ghcide to Stackage is useful, independent of anything else you've said. Perhaps you want to create another ticket to advocate including another option? It's certainly not an either/or situation - we can be on Stackage and have binaries, a Nix script, a homebrew installation etc.

**Comment by v217** [comment]
@ndmitchell Absolutely agree, just wanted to mention this option!

**Comment by ndmitchell** [comment]
Btw, its likely that specific forms of distribution (including binary) will be done through https://github.com/haskell/haskell-language-server rather than ghcide, and that project will bundle ghcide plus plugins.

**Comment by ndmitchell** [comment]
Once https://github.com/digital-asset/ghcide/pull/429 gets merged we'll be in a state to add it. Furthermore, it's just a few days before the next Stackage LTS cutoff, so that's convenient.

**Comment by ejconlon** [comment]
I can confirm that everything builds fine and all tests pass locally with lts-15.0. However, a new release (0.2.0?) is needed to allow `stack install ghcide` to work outside the project.

**Comment by ndmitchell** [comment]
#464 was caused by not yet being in Stackage.

**Comment by jamesdbrock** [comment]
You know, it would make a huge amount of sense for `ghcide` to be in Stackage.

In the top directory of any `stack.yaml`-based project, you could just `$ stack exec --package ghcide ghcide -- --lsp` and then `ghcide` would be locally compiled with the same version of GHC that project uses, and executed.

**Comment by jneira** [comment]
I think we could merge this with https://github.com/haskell/haskell-language-server/issues/400
